### ISSUE TITLE
[BUG] AGPL license is sometimes not reported as restrictive

### ISSUE THREAD

**Issue by msrb** [fabric8-analytics/fabric8-analytics-vscode-extension#341]
I have following pom.xml file:

```
<project>
    <modelVersion>4.0.0</modelVersion>
    <groupId>g</groupId>
    <artifactId>a</artifactId>
    <version>0.0.1</version>

    <dependencies>
        <dependency>
            <groupId>org.json</groupId>
            <artifactId>json</artifactId>
            <version>20190722</version>
        </dependency>

        <dependency>
            <groupId>com.itextpdf</groupId>
            <artifactId>itextpdf</artifactId>
            <version>5.5.13</version>
        </dependency>

        <!-- <dependency>
            <groupId>com.google.code.gson</groupId>
            <artifactId>gson</artifactId>
            <version>2.8.5</version>
        </dependency> -->
    </dependencies>

    <licenses>
        <license>
            <name>Apache License, Version 2.0</name>
            <url>https://www.apache.org/licenses/LICENSE-2.0.txt</url>
            <distribution>repo</distribution>
            <comments>A business-friendly OSS license</comments>
        </license>
    </licenses>
</project>
```

com.itextpdf:itextpdf is licensed under restrictive AGPL license, however analysis shows 0 restrictive licenses.

When I add another package (com.google.code.gson:gson:2.8.5) to my stack, it suddenly shows 1 restrictive license. Although there doesn't seem to be a way how to pin point which of the dependencies is the one with restrictive license.

**VSCode(please complete the following information):**
 - OS: linux
 - VSCode version 1.37.1
 - Dependency Analytics Version 0.0.12

**Comment by msrb** [comment]
Also it's a bit strange that JSON license (org.json:json) is not reported. The license is effectively non-free.

**Comment by msrb** [comment]
The result without `com.google.code.gson:gson:2.8.5` in the stack:

![Screenshot from 2019-09-10 13-02-56](https://user-images.githubusercontent.com/3759039/64608710-939ef200-d3cb-11e9-9e41-526a98beed00.png)

**Comment by msrb** [comment]
The result with `com.google.code.gson:gson:2.8.5` in the stack:

![Screenshot from 2019-09-10 13-03-40](https://user-images.githubusercontent.com/3759039/64608773-af09fd00-d3cb-11e9-8327-ddc11ddeb133.png)

**Comment by yzainee** [comment]
@msrb @arajkumar This is not an issue. The behaviour is correct. When there are 2 deps, it cannot decide which dep is an outlier. When another dep is added, having a permissive lic, then it shows the restrictive lic. However, Michal has a  point that the UI is a bit misleading in providing this information. Probably we can think about this.

**Comment by yzainee** [comment]
https://jira.coreos.com/browse/APPAI-906

**Comment by msrb** [comment]
Yes, the UI is super confusing. It says "flag any unknown/restricted license(s) present". That's clearly not the case. Based on what Yusuf said, It seems like underlying backend service provides information about "outlier licenses" and UI misinterprets the information. If we accept the fact that the number is correct, then the way it is presented in UI is very misleading.

**Comment by yzainee** [comment]
The only thing i would like to add here is that it gives the outliers only in case, where they have a restrictive license.

**Comment by arajkumar** [comment]
@yzainee , Can you please close this if it is not an issue?

**Comment by yzainee** [comment]
This is not an issue.
